src/jarallax.esm.js

```diff
@@ -143,6 +197,7 @@ class Jarallax {
             imgRepeat: 'no-repeat', // supported only for background, not for <img> tag
             keepImg: false, // keep <img> tag in it's default place
             elementInViewport: null,
+            scrollableContainer: null,
```

Comment:
We should update the `README.md` file with this new option and explain its purpose / usage.

Reply:
**Only a suggestion, feel free to ignore:** We could consider auto detecting this rather than requesting it as an option, jQuery implemented a function for `scrollParent()`: https://api.jqueryui.com/scrollParent/. Here is the source from jQuery UI: https://github.com/jquery/jquery-ui/blob/74f8a0ac952f6f45f773312292baef1c26d81300/ui/scroll-parent.js

This could result in unexpected behavior so the additive change is better in this case.

Reply:
I thought about this and I agree that it could result in some unexpected behavior.

Reply:
> We should update the `README.md` file with this new option and explain its purpose / usage.

Updated.